hack/jenkins/windows_integration_test_hyperv.ps1

```diff
@@ -17,6 +17,9 @@ gsutil.cmd cp gs://minikube-builds/$env:MINIKUBE_LOCATION/minikube-windows-amd64
 gsutil.cmd cp gs://minikube-builds/$env:MINIKUBE_LOCATION/e2e-windows-amd64.exe out/
 gsutil.cmd cp -r gs://minikube-builds/$env:MINIKUBE_LOCATION/testdata .
 
+
+taskkill /im minikube-windows-amd64.exe /f
+Remove-Item -Recurse -Force C:\Users\jenkins\.minikube
 ./out/minikube-windows-amd64.exe delete
 
```

Comment:
Why not remove the folder after minikube deletes and skip the `taskkill`? Why do we need to taskkill?